src/infrastructure/utils/ffmpeg_helper.py

```
@@ -314,19 +317,208 @@ async def has_valid_video_stream(
+        return expire_ts is not None and expire_ts > now_ts
+
+    @staticmethod
+    def _gc_transcode_cache(
+        cache_root: Path,
+        *,
+        suffixes: tuple[str, ...],
+        now_ts: float,
+        cache_ttl_seconds: int,
+        skip_paths: set[Path] | None = None,
+    ) -> int:
+        """按 TTL 清理专用转码 cache，跳过当前 digest 的 legacy 文件。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)